src/tmx/TmxUtils/src/simulation/SimulationEnvUtils.h

```diff
@@ -31,6 +31,11 @@ namespace tmx::utils::sim{
          * in SIMULATION MODE for connecting to CDASim.
          */
         constexpr inline static const char *SIMULATION_REGISTRATION_PORT = "SIMULATION_REGISTRATION_PORT";
+        /**
+         * @brief sensors file location
+        */
+        constexpr inline static const char *SENSOR_JSON_FILE_PATH = "SENSOR_JSON_FILE_PATH";
```

Comment:
What was your reasoning for including this as an environment variable instead of a plugin configuration similar to location. Just curious, I think we need to define a clear line between what we use as environment variables and what ew configure in the plugin. Simulation configurations are good to have in as environment variables because they do not require configuration and can be read easily at startup. They are also unlikely to change dynamically. I think that makes sense for the sensor configuration file as well.

Reply:
Did you answer your own question? Yeah, I agree that the file path for sensors shall not change constantly.

Reply:
Yeah just wanted to bring it up, that we give thought to what is an environment variable and what is a configuration parameter.